csharp/ql/test/TestUtilities/PrettyPrintModels.ql

```diff
@@ -0,0 +1,18 @@
+/**
+ * @kind test-postprocess
+ */
+
+import codeql.dataflow.test.ProvenancePathGraph
```

Comment:
Needs to be rebased now that https://github.com/github/codeql/pull/17179 has been merged.